src/main/java/io/debezium/connector/yugabytedb/YBClientUtils.java

```diff
@@ -229,4 +232,54 @@ public static OpId getOpIdFromGetTabletListResponse(GetTabletListToPollForCDCRes
     // Return null if no match is found, ideally this shouldn't happen in any case
     return null;
   }
+
+  /**
+   * Check whether the passed stream ID in the connector configuration has before image enabled.
```

Comment:
We should do this in the YB*COnnector and send that as a boolean prop to each task.